library/ziti.c

```diff
@@ -46,6 +59,9 @@ struct nf_init_req {
     void* init_ctx;
 };
 
+#define ZITI_MD_MAX_SIZE_256 32  /* longest known is SHA256 or less */
```

Comment:
these not needed?

Reply:
I will delete from here... as they were relocated to ziti_enroll.c

Reply:
ununsed imports above (mbedtls/*)